src/test/java/org/example/c03/ontrolFlow/PrimeNumberCounterTest.java

```diff
@@ -0,0 +1,19 @@
+package org.example.c03.ontrolFlow;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.example.c03.controlFlow.PrimeNumberCounter;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class PrimeNumberCounterTest {
+    @Test
+    public void given3and9ShouldReturn3PrimeNumber() {
```

Comment:
Fix name : givenLower3Upper9ThenReturn3PrimeNumbers

Reply:
Done

Reply:
PrimeNumbers no PrimeNumber

Reply:
done

Reply:
I was referring to the test name. Its still not good

Reply:
done. thanks